index.html

```diff
@@ -1956,39 +1956,9 @@ <h3>Securing Verifiable Credentials</h3>
 mutually exclusive.
         </p>
         <p>
-<a>Verifiable credentials</a> and <a>verifiable presentations</a> MUST be secured by at least
-one securing method.
+<a>Verifiable credentials</a> MUST be secured by at least one securing method.
         </p>
-        <p>
-Methods of securing <a>verifiable credentials</a> or
-<a>verifiable presentations</a> that embed a proof in the data model MUST use
-the <code>proof</code> <a>property</a>.
-        </p>
-        <p>
-Methods of securing <a>verifiable credentials</a> or <a>verifiable
-presentations</a> that use an external proof MAY use the <code>proof</code>
-<a>property</a>.
-        </p>
-        <dl>
-          <dt><var>proof</var></dt>
```

Comment:
as proof is an attribute of data integrity secured conforming documents, it should be defined in the data integrity spec.

Reply:
Proof isn't specific to data integrity, but rather data integrity is a particular implementation an embedded proof. This is similar to "credentialStatus", "evidence", etc. So I think any changes in this PR that treat it differently from that should probably be adjusted back / removed.